platforms/Windows/shared/shared.wxs

```diff
@@ -52,6 +52,16 @@
                 <Directory Id="_usr_share_docc" Name="docc">
                   <Directory Id="_usr_share_docc_render" Name="render" />
                 </Directory>
+                <Directory Id="_usr_share_pm" Name="pm">
```

Comment:
Why is this in shared? I think that this should be in cli.wxs

Reply:
No particular reason, fixed it